src/options/UnitSettingsForm.tsx

```diff
@@ -0,0 +1,38 @@
+import { useLocale } from '../common/i18n';
+
+type Props = {
+  selectedUnits: 'metric' | 'imperial';
+  onChange: (currency: string) => void;
+};
+
+export function UnitSettingsForm(props: Props) {
+  const { t } = useLocale();
+
+  const options = [
+    ['metric', t('options_units_metric_label')],
+    ['imperial', t('options_units_imperial_label')],
+  ];
+
+  return (
+    <>
+      <label for="preferredUnits">{t('options_units_label')}</label>
```

Comment:
We should probably add a `NewBadge` in here somewhere, e.g. `<NewBadge expiry={new Date('2024-09-30')} />`